docs/source/workflow.rst

```diff
@@ -1,6 +1,44 @@
 Cooking: How it all comes together
 ==================================
 
+Ways That PaaSTA Can Run Services
+---------------------------------
+
+Long Running Services
+^^^^^^^^^^^^^^^^^^^^^
+
+Long running services are are processes that are expected to run continuously
+and usually have the same process id throughout. PaaSTA uses
+`Marathon <yelpsoa_configs.html#marathon-clustername-yaml>`_ to configure how these
+services should run.
+
+These services often serve network traffic, usually HTTP. PaaSTA integrates with
+SmartStack to make it easy for others to discover these long running services, or
+operators can use alternative service discovery methods to interact with them.
+
+Alternative names: workers, daemons, batch daemons, servers
+
+Scheduled Jobs
+^^^^^^^^^^^^^^
+
+Scheduled tasks are those tasks that are periodically run, and are not expected
+to run continously. Due to their ephemeral nature, they often do not expose a TCP port.
+
+PaaSTA uses `Chronos <yelpsoa_configs.html#chronos-clustername-yaml>_` to define
+how these scheduled jobs should be executed, and how often they should run.
```

Comment:
s/how these scheduled jobs should be executed/what command these scheduled jobs should execute/